tests/release/test_fixture_args.py

```diff
@@ -0,0 +1,96 @@
+import os
+import subprocess
+from pathlib import Path
+
+import pytest
+
+from pseudopeople.interface import (
+    generate_american_community_survey,
+    generate_current_population_survey,
+    generate_decennial_census,
+    generate_social_security,
+    generate_taxes_1040,
+    generate_taxes_w2_and_1099,
+    generate_women_infants_and_children,
+)
+from tests.release.conftest import (
+    DEFAULT_ENGINE,
+    DEFAULT_STATE,
+    DEFAULT_YEAR,
+    FULL_USA_FILEPATH,
+    RI_FILEPATH,
+    _parse_dataset_params,
+)
+
+
+@pytest.fixture()
+def check_subprocess_environment() -> None:
+    if "RUNNING_AS_SUBPROCESS" not in os.environ:
+        pytest.skip("Skipping this test because it's not running as a subprocess")
+
+
+EXPECTED_PARAMETERS = {
```

Comment:
I don't understand what this object is doing.

Reply:
We want to test that the parameters which are passed as flags when calling pytest are properly converted to the parameters that we pass to the generate_dataset function. Each tested parameter combination is uniquely identified by the combination's dataset name. The dictionary keys are the dataset names identifying the combination of parameters, and the elements are the converted parameter arguments.